app/models/legacy/server.rb

```diff
@@ -1,29 +1,17 @@
 class Legacy::Server
 
-  def update(payload)
-    begin
-      RestClient.post(url, payload, headers)
-    rescue => e
-      Rails.logger.debug "ERROR! => update not sent to legacy"
-      raise e
-    end
+  def update(path, payload)
+    url = URI.join(host, path).to_s
+    LegacyUpdateWorker.perform_async(url, payload, headers)
   end
 
   private
 
-  def url
-    URI.join(host, path).to_s
-  end
-
   def host
-    ENV['LEGACY_HOST']
-  end
-
-  def path
-    "/api/3/action/package_patch"
+    ENV.fetch('LEGACY_HOST')
```

Comment:
blank fetch will just return the same, surely?

Reply:
I want this to fail if the `LEGACY_HOST` key is absent. In order to avoid passing a nil` all the way down to `RestCient#post`.

Reply:
Ah, cool.